Demo/Demo/FeatureViewControllers/PaymentButtonCustomizationViewController.swift

```diff
@@ -0,0 +1,213 @@
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            stackViewHeight,
+            stackView.widthAnchor.constraint(
+                equalTo: scrollView.frameLayoutGuide.widthAnchor
+            ),
+            stackView.leadingAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.leadingAnchor
+            ),
+            stackView.trailingAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.trailingAnchor
+            ),
+            stackView.topAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.topAnchor
+            ),
+            stackView.bottomAnchor.constraint(
+                equalTo: scrollView.contentLayoutGuide.bottomAnchor
+            )
+        ]
+    }
+
+    // MARK: - Target / Actions
+
+    @objc private func reloadSmartPaymentButton(_ sender: UISegmentedControl) {
+        if sender == fundingPicker {
+            reloadColorPicker()
+        }
+        paymentButton.removeFromSuperview()
+
+        paymentButton = setupPaymentButton()
+        stackView.addArrangedSubview(paymentButton)
+    }
+
+    func setShadow(
+        on button: PaymentButton,
+        color: UIColor = .black,
+        offset: CGSize = CGSize(width: 0, height: 1),
+        opacity: Float = 0.3,
+        radius: CGFloat = 1.0
+    ) {
+        button.layer.shadowOpacity = opacity
+        button.layer.shadowRadius = radius
+        button.layer.shadowOffset = offset
+        button.layer.shadowColor = color.cgColor
+    }
+
+    private func reloadColorPicker() {
+        let funding = PaymentButtonFundingSource.allCases()[fundingPicker.selectedSegmentIndex]
+        let segments = colorPickerSegments(with: funding)
+
+        colorPicker.removeAllSegments()
+        for segment in segments {
+            colorPicker.insertSegment(
+                withTitle: segment,
+                at: colorPicker.numberOfSegments,
+                animated: false
+            )
+        }
+        colorPicker.selectedSegmentIndex = 0
+    }
+
+    private func colorPickerSegments(with funding: PaymentButtonFundingSource) -> [String] {
+        switch funding {
+
+        case .payPal:
+            return PayPalButton.Color.allCasesAsString()
+
+        case .payLater:
+            return PayPalPayLaterButton.Color.allCasesAsString()
+
+        case .credit:
+            return PayPalCreditButton.Color.allCasesAsString()
+        }
+    }
+
+    private func setupPaymentButton() -> PaymentButton {
+        let fundingSource = PaymentButtonFundingSource.allCases()[fundingPicker.selectedSegmentIndex]
+        let edges = PaymentButtonEdges.allCases()[edgesPicker.selectedSegmentIndex]
+        let size = PaymentButtonSize.allCases()[sizePicker.selectedSegmentIndex]
+
+        let paymentButton: PaymentButton
+
+        switch fundingSource {
+
```

Comment:
Nit: typically we would have no space between switch and the first case and then spaces between each case like so:
```
switch thing {
case 1:
    return 1

case 2:
    return 2

case 3:
    return 3
}